lib/modules/apostrophe-doc-type-manager/index.js

```diff
@@ -59,6 +59,11 @@ module.exports = {
             value: 'certainUsers',
             label: 'Certain People',
             showFields: [ '_viewGroups', '_viewUsers' ]
+          },
+          {
+            value: 'excludeCertainUsers',
```

Comment:
It is unclear from the label that logged-out users are also excluded (although it certainly wouldn't do the job if they weren't). This needs a well written help property, at least.